### PR TITLE
HTML update

### PR DESCRIPTION
**Pull request by bedroesb**
- RO-crate logo in svg
- more padding
- lists are nicer printed
- support for CreativeWorkStatus and publisher

image

### CODE REVIEW

File: rocrate/templates/preview_template.html.j2 [hunk 1]
```diff
@@ -81,9 +141,21 @@
...
+  {% if crate.CreativeWorkStatus %}
+    <dt>CreativeWorkStatus</dt>
+    <dd>{{ crate.CreativeWorkStatus }}</dd>
+  {%endif %}
   {% if crate.creator %}
     <dt>Creator</dt>
+    {% if crate.creator is string %}
     <dd>{{ crate.creator }}</dd>
+    {%else %}
+    <dd>{{ crate.creator|join(', ') }}</dd>
```
Comment (ieguinoa) [inline rocrate/templates/preview_template.html.j2:157]: here you are checking if crate.creator is a string and in the else condition you assume is a list but in practice the options are string, list or ContextualEntity object (or a combination of these). In the test_write case that fails the creator is set to a Contextualentiy:
new_person = crate.add_person('001', {'name': 'Lee Ritenour'})

File: rocrate/templates/preview_template.html.j2 [hunk 2]
```diff
@@ -61,17 +121,17 @@
...
   {% if crate.license %}
     <dt>License</dt>
     <dd>{{ crate.license }}</dd>
   {%endif %}
   {% if crate.keywords %}
     <dt>Keywords</dt>
+    {% if crate.keywords is string %}
     <dd>{{ crate.keywords }}</dd>
+    {%else %}
+    <dd>{{ crate.keywords|join(', ') }}</dd>
```
Comment (ieguinoa) [inline rocrate/templates/preview_template.html.j2:133]: same here

Comment (bedroesb) [reply rocrate/templates/preview_template.html.j2:133]: Can keywords be objects?

Comment (ieguinoa) [reply rocrate/templates/preview_template.html.j2:133]: you can assume so. there is no fixed rule in the code but it makes sense

**Comment by bedroesb** [comment]
will do some improvements for person objects

**Comment by bedroesb** [comment]
image



@ieguinoa I think we are there now

**Pull request merged**